### ISSUE TITLE
这是什么操作？

### ISSUE THREAD

**Issue by xbojin** [PolarisShi/distillation#3]
with torch.no_grad():
    for data in trainloader:
        images, labels = data

softset = torch.utils.data.TensorDataset(images, labels, soft_target)

相当于读了一个batch_size的images放入了新的数据集，很迷。
